beetmoverscript/templates/firefox_nightly.yml

```diff
@@ -129,6 +129,12 @@ mapping:
       destinations:
         - {{ upload_date }}-{{ branch }}/mozharness.zip
         - latest-{{ branch }}/mozharness.zip
+    buildhub.json:
+      s3_key: firefox-{{ version }}.{{ locale }}.{{ platform }}.buildhub.json
+      destinations:
+        - {{ upload_date }}-{{ branch }}/firefox-{{ version }}.{{ locale }}.{{ platform }}.buildhub.json
+        - latest-{{ branch }}/firefox-{{ version }}.{{ locale }}.{{ platform }}.buildhub.json
+        - latest-{{ branch }}-l10n/firefox-{{ version }}.{{ locale }}.{{ platform }}.buildhub.json
```

Comment:
Not sure why we have three entrie for "firefox_nightly" but not the same for "fennec_nightly". However, I think it's useless. As far as I can tell from Peter's [comments](https://bugzilla.mozilla.org/show_bug.cgi?id=1443873#c16)  they are going to consume the artifacts as they are, from the entire dir structure / release so I don't think it's a problem.